tools/wptrunner/wptrunner/testloader.py

```diff
@@ -546,9 +546,9 @@ def group_metadata(cls, state):
     def group(self):
         if not self.current_group.group or len(self.current_group.group) == 0:
             try:
-                self.current_group = self.test_queue.get(block=True, timeout=5)
+                self.current_group = self.test_queue.get_nowait()
             except Empty:
-                self.logger.warning("Timed out getting test group from queue")
+                self.logger.warning("No test group in the queue")
```

Comment:
I think it's still better to add one more sentinal for each test runner, otherwise we see this warning always.

Reply:
We could, but then a sentinel would represent a call site and not just a worker, and we would need to keep the sentinel count accurate.

Should we just get rid of the sentinels? An empty queue already represents the null test group, but there's no waiting penalty without blocking now. Then we can also get rid of the log, since an empty queue is part of normal operation.

Reply:
Sure, let's do that.